### DISCUSSION TITLE
How to display list data using a static array in Filament without using Eloquent.

### DISCUSSION THREAD

**Discussion by ZicLemon** [filamentphp/filament#12165] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

How to display list data using a static array in Filament without using Eloquent.

**Comment by leandrocfe** [comment]
https://filamentphp.com/community/how-to-consume-an-external-api-with-filament-tables

**Reply by killjin** [reply]
api with param, how to get rows by params?
eg: api?category=$var
